### PR TITLE
Move request to http package.

### PR DESCRIPTION
**Pull request by markstory**
Add an alias for backwards compatibility. I've not moved the tests yet as I wanted to ensure that backwards compatibility was not lost.

Refs #9325

### CODE REVIEW

**Comment by ADmad** [comment]
Nice. Would it possible to also create a split repo for the Http namespace and provide it as a independent package?

**Comment by markstory** [comment]
@ADmad I want to. Once the response is done, the subsplit will create something useful.

File: src/Http/ServerRequest.php [hunk 1]
```diff
@@ -0,0 +1,2062 @@
...
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         2.0.0
```
Comment (ADmad) [inline src/Http/ServerRequest.php]: Should this be 3.4.0?

**Comment by ADmad** [comment]
Test failing.

**Comment by markstory** [comment]
@ADmad Tests fixed now.

File: tests/TestCase/Error/DebuggerTest.php [hunk 2]
```diff
@@ -272,7 +272,7 @@ public function testExportVar()
 	theme => null
 	hasRendered => false
 	uuids => []
-	request => object(Cake\Network\Request) {}
+	request => object(Cake\Http\ServerRequest) {}
 	response => object(Cake\Network\Response) {}
```
Comment (dereuromark) [inline tests/TestCase/Error/DebuggerTest.php]: Should the response be ServerResponse then for consistency in naming?

Comment (markstory) [inline tests/TestCase/Error/DebuggerTest.php]: It will in due time.

Comment (ADmad) [inline tests/TestCase/Error/DebuggerTest.php]: The Psr interface name is ResponseInterface and Diactoros class name is also Response. So neither uses "Server" prefix.

Comment (dereuromark) [inline tests/TestCase/Error/DebuggerTest.php]: The Cake names should still be reasonably consistent in itself.

**Comment by lorenzo** [comment]
Nice!

**Pull request merged**